Gemfile

```diff
@@ -1,153 +1,105 @@
-  gem 'spring', '~> 1.7'
-  gem 'spring-watcher-listen', '~> 2.0.0'
-  # gem 'xray-rails'
-end
-
-gem 'bulkrax', git: 'https://github.com/samvera-labs/bulkrax.git', ref: '817e4bb'
-
+gem 'active_elastic_job', github: 'active-elastic-job/active-elastic-job', ref: 'ec51c5d9dedc4a1b47f2db41f26d5fceb251e979', group: %i[aws]
+gem 'activerecord-nulldb-adapter'
+gem 'addressable', '2.8.1' # remove once https://github.com/postrank-labs/postrank-uri/issues/49 is fixed
 gem 'allinson_flex', git: 'https://github.com/samvera-labs/allinson_flex.git'
+gem 'apartment'
+gem 'aws-sdk-sqs', group: %i[aws]
 gem 'blacklight', '~> 6.7'
 gem 'blacklight_oai_provider', '~> 6.1', '>= 6.1.1'
-
-# lock to graph indexing update commit
-gem 'hyrax', git: 'https://github.com/samvera/hyrax.git', branch: '3.x-stable'
-# we need redlock to be at a lower version that Hyrax 3.x defines, otherwise it causes fileset attachment errors
-# ref: https://github.com/samvera/hyrax/blob/d83b8392e6a30c6ad4915a2f13d174de199afb07/hyrax.gemspec#L78
-gem 'redlock', '~>1.2.2'
-
-gem 'hyrax-iiif_av', git: 'https://github.com/samvera-labs/hyrax-iiif_av.git', branch: 'utk-hyku-with-hyrax-3'
-gem 'iiif_manifest', git: 'https://github.com/samvera/iiif_manifest.git', ref: 'e5d8a2d'
-gem 'iiif_print', git: 'https://github.com/scientist-softserv/iiif_print.git', branch: 'main'
-
 gem 'bolognese', '>= 1.9.10'
-gem 'hyrax-doi', git: 'https://github.com/samvera-labs/hyrax-doi.git'
-gem 'postrank-uri', '>= 1.0.24'
-gem 'rsolr', '~> 2.0'
-
+gem 'bootstrap-datepicker-rails'
+gem 'bulkrax', git: 'https://github.com/samvera-labs/bulkrax.git', ref: '817e4bb'
+gem 'byebug', group: %i[development test]
+gem 'capybara', group: %i[test]
+gem 'capybara-screenshot', '~> 1.0', group: %i[test]
+gem 'carrierwave-aws', '~> 1.3', group: %i[aws test]
+gem 'cocoon'
+gem 'codemirror-rails'
+gem 'coffee-rails', '~> 4.2' # Use CoffeeScript for .coffee assets and views
+gem 'database_cleaner', group: %i[test]
 gem 'devise'
 gem 'devise-guests', '~> 0.3'
 gem 'devise-i18n'
 gem 'devise_invitable', '~> 1.6'
-
-gem 'apartment'
-gem 'is_it_working'
-gem 'rolify'
-
-gem 'flipflop', '~> 2.3'
+gem 'dry-monads', '~> 1.4.0' # Locked because 1.5.0 was not compatible with Hyrax v.3.4.2
+gem 'easy_translate', group: %i[development]
+gem 'factory_bot_rails', group: %i[test]
+gem 'fcrepo_wrapper', '~> 0.4', group: %i[development test]
+gem 'flipflop', '~> 2.6.0' # waiting for hyrax 4 upgrade
+gem 'flutie'
+gem 'good_job', '~> 2.99'
+gem 'hyrax', git: 'https://github.com/samvera/hyrax.git', branch: '3.x-stable'
+gem 'hyrax-doi', github: 'samvera-labs/hyrax-doi', branch: 'main'
+gem 'hyrax-iiif_av', github: 'samvera-labs/hyrax-iiif_av', branch: 'utk-hyku-with-hyrax-3'
+gem 'i18n-debug', require: false, group: %i[development test]
+gem 'i18n-tasks', group: %i[development test]
+gem 'iiif_manifest', git: 'https://github.com/samvera/iiif_manifest.git', ref: 'e5d8a2d'
+gem 'iiif_print', github: 'scientist-softserv/iiif_print', branch: 'main'
```

Comment:
I know these already existed, but why do we sometimes use `git:` and at other times use `github:`?

Reply:
also, not asking for it to be changed in this pr. just curious.

Reply:
@alishaevn Bundler has supported both git: and github: from the first commit that introduced either function (back in pre 1.0 of bundler). However for several years github: would use an insecure method for pulling the git repos (git://) and that could be spoofed or read. So everyone changed the github: to the longer git: (there are, of course non-github places you can store git repos). THEN a patch came out you could apply. basically you redefined github: at the top of the gemfile to use the https:// instead of git://. Finally with the release of Bundler 2.0 https:// became the default again.